HotelBooking/Controllers/reservation_controller.py

```diff
@@ -36,7 +36,7 @@ def get_reservations(self, user_id=None) -> List[Reservation]:
     def get_stay_history(self, user_id=None) -> List[Reservation]:
         return self.reservation.get_reservations_with_status("CLOSED", user_id)
 
-    def reserve_room(self, room_id: str, customer_id: int, start_date, duration: int, status: str = None):
+    def reserve_room(self, room_id: str, customer_id: int, start_date, duration: int, is_accessibility_requested: int, status: str = None):
```

Comment:
`is_accessibility_requested: int` sounds like a boolean variable, why is it defined as an int here?

Reply:
No support for Boolean